modules/user-data/templates/06_application_insights_config.sh.tpl

```diff
@@ -6,12 +6,93 @@ echo "###########################################"
 echo "#    Configuring Application Insights     #"
 echo "###########################################"
 
+RECORD_NAME=$(cat /tmp/node_name)
 
 # Overrides the config file
 cat <<-EOF > /opt/graphdb/applicationinsights.json
 {
-  "connectionString": "${appi_connection_string}"
+    "role": {
+        "name": "$RECORD_NAME"
+      },
+    "connectionString": "${appi_connection_string}",
+    "sampling": {
+      "percentage": ${appi_sampling_percentage}
+  },
+    "instrumentation": {
+        "logging": {
+            "level": "${appi_logging_level}"
+        }
+    },
+    "preview": {
+        "captureLogbackMarker": true,
+        "captureLog4jMarker": true,
+        "captureLogbackCodeAttributes": true,
+        "sampling": {
+            "overrides": [
+              {
+                "telemetryType": "dependency",
+                "percentage": ${appi_dependency_sampling_override}
+              },
+              {
+                "telemetryType": "request",
+                "percentage": 0,
```

Comment:
I think this could be useful for troubleshooting so maybe add a configuration with 0 as the default.

Reply:
IMHO logging these requests brings no value at all, that's why I've set them to 0.
For example why would we ever need to see get requests for `js|css|html|woff2|png|svg|gif|favicon`

Also requests for `cluster|namespaces|status|locations.*|license.*|acl|monitor.*|autocomplete.*|cluster.*|saved-queries|security.*|version.*|all|connectors.*|info.*|rdfrank.*|/sql-views.*` won't bring us much information for debugging or performance gathering

Reply:
I was talking about the gRPC's server filter.

Reply:
done